packages/base-dao/test/baseTestForDAOs.js

```diff
@@ -258,12 +258,12 @@ function randomValuesByType(dataType) {
     // Hardcoded values, e.g. enum.
     return [dataType[0], dataType[1]];
   }
-  const timeZoneSuffix = '';
+
   switch (dataType) {
     case 'date':
-      return [`${momentRandom().format('YYYY-MM-DD')}${timeZoneSuffix}`, `${momentRandom().format('YYYY-MM-DD')}${timeZoneSuffix}`];
+      return [`${momentRandom().format('YYYY-MM-DD[T]00:00:00.000Z')}`, `${momentRandom().format('YYYY-MM-DD[T]00:00:00.000Z')}`];
```

Comment:
what will happen if some entity run a test on a field whose time component matters (that is, date and not dateonly)? e.g. calendar booking...

Reply:
fixed to format HH:mm:ss:sssZ